web/src/main/resources/swagger/swagger_en.properties

```diff
@@ -8,6 +8,7 @@ swagger.mscore.external.geotaxonomies=retrieves the geographic taxonomies relate
 swagger.mscore.external.institution.relationships=returns the relationships related to the institution
 swagger.mscore.institutions=Gets institutions filtering by taxCode and/or subunitCode
 swagger.ms-core.scheduler.api.start=Service to start scheduler to resend old messages to DL
+swagger.ms-core.scheduler.api.start.users=Service to resend old user onboardings to DL
```

Comment:
do we send only to datalake? or to kafka queue for all consumers?.....we should also add some detail about page and size, which is the order about list based for page and size?

Reply:
Actually to all the consumers connected to the queue

Reply:
ok so you could remove "to DL" and add that we send to a queue ScUsers....in addition, you could add some detail about page and size in which order the list is retrieved